PythonAPI/setup.py

```diff
@@ -12,26 +13,36 @@
 
 import numpy as np
 
-ext_modules = [
-    Extension(
-        'pycocotools._mask',
-        sources=['./common/maskApi.c', 'pycocotools/_mask.pyx'],
-        include_dirs = [np.get_include(), './common'],
-        extra_compile_args=['-Wno-cpp', '-Wno-unused-function', '-std=c99'],
-    )
-]
+if platform.system() == 'Windows':
+    ext_modules = [
+        Extension(
+            'pycocotools._mask',
+            sources=['./common/maskApi.c', 'pycocotools/_mask.pyx'],
+            include_dirs=[np.get_include(), './common'],
+            extra_compile_args=[],
+        )
+    ]
+else:
+    ext_modules = [
+        Extension(
+            'pycocotools._mask',
+            sources=['./common/maskApi.c', 'pycocotools/_mask.pyx'],
+            include_dirs=[np.get_include(), './common'],
+            extra_compile_args=['-Wno-cpp', '-Wno-unused-function', '-std=c99'],
```

Comment:
if this line is the only difference, you can move the `if windows ...` to the right hand side of this line

Reply:
>  I'm not sure it works the same on windows. Please double-check the whole process from the beginning when you finalize the PR

I executed the complete process again and its completely working fine over both Windows and Linux with a prior installation of Cython on Windows.  

> if this line is the only difference, you can move the if windows ... to the right hand side of this line

Done! Thanks!